conf/development.ini

```diff
@@ -58,6 +58,7 @@ secret_key: notverysecretafterall
 sqlalchemy.url: sqlite:///.h.db
 
 jinja2.extensions: h.jinja_extensions:IncludeRawExtension
+jinja2.filters: json = json.dumps
```

Comment:
It seems odd to me that this is app configuration. For example, you haven't currently added this to `test.ini`, so if the tests at any point in the future try and render `features.coffee` (or any other template which in the future uses the `json` filter), they'll explode.

Perhaps this (and, actually, the extensions configuration) should move into the code?